src/Upgrader/Business/Git/CommandResolver/UpdateIndexCommand.php

```diff
@@ -28,6 +28,6 @@ public function isIndexOutdated(): bool
     {
         $commandResultDto = $this->exec($this->getCommand() . ' ' . self::REFRESH_FLAG);
 
-        return boolval($commandResultDto->getResultCode());
+        return (bool)$commandResultDto->getResultCode();
```

Comment:
My question still stands. Why is this done here? Isnt the error code inverse to the boolean value?

Reply:
'(bool)' we need for avoid PhpStan error: Method Upgrader\Business\Git\CommandResolver\UpdateIndexCommand::isIndexOutdated() should return bool but returns int.

Reply:
If we have error code (1, 2 ,15 .... ) from getResultCode() it's mean `git update-index --refresh` failed as result we know that project has uncommitted changes

Reply:
OK